tasks/init

```diff
@@ -12,10 +12,12 @@
 
 vp_init_cache=$HOME/.vault-playground/init_dumps
 vp_network_name=vp
+vp_vault_shares=5
+vp_vault_threshold=3
 
 # Environment variables needed by this script defaulted for local use
 : "${VP_AUTO_INIT:=true}" # If true, after launching Vault also runs init, caches the resulting keys, and automatically unseals Vault
-: "${VP_HA_MODE:=true}" # If true will spin up a second Vault server ready to run as a standby. Unless AUTO_INIT_VALUE is true this server will need to be manually unsealed to enter standby mode
+: "${VP_VAULT_CLUSTER_SIZE:=2}" # If true will spin up a second Vault server ready to run as a standby. Unless AUTO_INIT_VALUE is true this server will need to be manually unsealed to enter standby mode
```

Comment:
maybe comment should read 'If greater than 1'

Reply:
:+1: Aye dat left over comment is tricky